corehq/apps/userreports/tasks.py

```diff
@@ -170,7 +170,7 @@ def _run_report(backend_to_use):
             json_response["total_row"] = total_row
         return json_response
 
-    spec = get_document_or_not_found(ReportConfiguration, domain, report_config_id)
```

Comment:
It seems as if this task is only used in one place, [`ConfigurableReport.get_ajax()`](https://github.com/dimagi/commcare-hq/blob/871586315e7c155cf70382f4f91ff204fff7b188/corehq/apps/userreports/reports/view.py#L429-L429). `DocumentNotFound` exceptions are not caught, and the return result is not used.